docs/storage/flysystem.md

```diff
@@ -63,7 +63,7 @@ vich_uploader:
 
 **Note:**
 
-> At the moment, only version 3 of bundle is supported.
+> At the moment, no version of bundle is supported.
```

Comment:
```suggestion
> At the moment, only version 4 of bundle is supported.
```

Reply:
I would point out to exact version. Something like "Since VichUploaderBundle v1.17, version 4 of bundle is supported. Version 3 is supported until VichUploaderBundle v1.16